cdap-ui/app/features/hydratorplusplus/hydrator-modal.less

```diff
@@ -148,7 +148,7 @@
 
         .modal-body {
           padding: 0 10px;
-          overflow: auto;
+          overflow: hidden;
```

Comment:
Yeah. For some reason safari shows two levels of scroll bars one for the entire node config modal and the other for the tab contents. :|